src/line_cursor.ts

```diff
@@ -402,6 +402,49 @@ export class LineCursor extends Marker {
     return this.getRightMostChild(newNode);
   }
 
+  /**
+   * Get the current location of the marker.
```

Comment:
Current location of the _cursor_, not the marker, here and on line 421.

Reply:
Hah: description copied from superclass, but fair enough!